django_babel/extract.py

```diff
@@ -1,5 +1,20 @@
 # -*- coding: utf-8 -*-
-from django.template.base import Lexer, TOKEN_TEXT, TOKEN_VAR, TOKEN_BLOCK
+try:
+    from django.template.base import Lexer, TOKEN_TEXT, TOKEN_VAR, TOKEN_BLOCK
+except ImportError:
+    try:
+        # Django 1.8 moved most stuff to .base
+        from django.template.base import (
```

Comment:
Why nested try..excepts ? This won't work ?

```
try:
   from django.template.base import Lexer, TOKEN_TEXT, TOKEN_VAR, TOKEN_BLOCK
except:
￼   from django.template.base import Lexer, TokenType
￼   
   TOKEN_TEXT = TokenType.TEXT
￼   TOKEN_VAR = TokenType.VAR
￼   TOKEN_BLOCK = TokenType.BLOCK
```

Reply:
Sure thing, I don't why I missed it :(